plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2319,6 +2321,20 @@ elementgui.tool.stays_in_grid_when_crafting=Does item stay in crafting grid when
 elementgui.tool.damaged_on_crafting=<html>Damage item instead on crafting<br><small>Make sure to enable "stay in crafting grid" and that item is damageable
 elementgui.tool.tooltip_tip=<html>Special information about the tool:<br><small>Use comma to separate entries
 elementgui.tool.needs_a_name=Tool needs a name
+elementgui.villager_trade.add_profession_trades=Add profession trades
+elementgui.villager_trade.add_entry=Add trade entry
+elementgui.villager_trade.remove_profession_trades=Remove the profession trades
+elementgui.villager_trade.remove_entry=Remove this entry
+elementgui.villager_trade.profession_trades=Profession trades
+elementgui.villager_trade.profession=Profession
+elementgui.villager_trade.level=Level
+elementgui.villager_trade.price1=Price 1
+elementgui.villager_trade.price2=Price 2
+elementgui.villager_trade.sale1=Sale 1
```

Comment:
This could be just "Sale", I don't think Mojang will add a second sale slot anytime soon

Reply:
It's done.